src/AlignedSpans.jl

```diff
@@ -224,21 +287,19 @@ Note: `span` may be of any type which which provides methods for `AlignedSpans.s
     How can this be? Clearly `Second(60) > Second(30) + Nanosecond(1)`, so what is going on?
 
     To understand this, note that at 1/30Hz, the samples occur at 00:00, 00:30, 00:60, and so forth. The original input span
-    includes _two_ samples, the one at 00:00 and the one at 00:30. Rounding inward to include only samples that occur within the span means including both samples.
+    includes _two_ samples, the one at 00:00 and the one at 00:30. Rounding inward to include only samples that occur within the span means including both samples (see [`RoundFullyContainedSampleSpans`](@ref) for alternate behavior).
 
     When converting back to TimeSpans, AlignedSpans canonicalizes `AlignedSpan(1/30, 1, 2)` as `TimeSpan(0, Second(60))`,
     representing these two samples as the time from the first sample until just before the not-included sample-3 (which occurs at `Second(60)`), using that `TimeSpan`'s exclude their right endpoint.
 
     AlignedSpans could make a different choice to e.g. canonicalize samples to only add an additional nanosecond,
     but that has its own issue (e.g. `TimeSpan(AlignedSpan(sample_rate, 1, 2))` and `TimeSpan(AlignedSpan(sample_rate, 3, 4))` would not be consecutive).
     
+See also [`AlignedSpan(sample_rate, span, mode::RoundingModeFullyContainedSampleSpans)`](@ref).
 """
 function AlignedSpan(sample_rate, span, mode::SpanRoundingMode)
     first_index = start_index_from_time(sample_rate, span, mode.start)
     last_index = stop_index_from_time(sample_rate, span, mode.stop)
-    if last_index < first_index
-        throw(ArgumentError("No samples lie within `span`"))
-    end
```

Comment:
deleting this since we check in the inner constructor with a more informative error message